src/httpclient.c

```diff
@@ -2,15 +2,23 @@
 
 #include <corto/httpclient/httpclient.h>
 #include <curl/curl.h>
+
 #define INITIAL_BODY_BUFFER_SIZE (512)
 
+/* Logging Support */
 corto_buffer g_logBuffer;
 bool g_logSet = false;
-
 struct url_data {
     size_t size;
     char* buffer;
 };
+
+/* Local Configuration */
+static corto_threadKey TLS_ConnectTimeoutKey;
+static corto_threadKey TLS_TimeoutKey;
+#define DEFAULT_CONNECT_TIMEOUT 500
+#define DEFAULT_TIMEOUT 300 * 1000
```

Comment:
What is the difference between `DEFAULT_CONNECT_TIMEOUT` and `DEFAULT_TIMEOUT`? Why are they different?

Reply:
**DEFAULT_CONNECT_TIMEOUT**: Connection phase timeout. Resolving and connecting to endpoint. 

**DEFAULT_TIMEOUT**: Transfer operation timeout.